README.md

```diff
@@ -41,11 +41,13 @@ or, convert any IO:
 from allotropy.parser_factory import Vendor
 from allotropy.to_allotrope import allotrope_from_io
 
-with open("filename.txt") as f:
-    asm_schema = allotrope_from_io(f, Vendor.MOLDEV_SOFTMAX_PRO)
+file_name = "filepath.txt"
+
+with open(file_name) as f:
+    asm_schema = allotrope_from_io(f, file_name, Vendor.MOLDEV_SOFTMAX_PRO)
 
 bytes_io = BytesIO(file_stream)
```

Comment:
this example here is two separate examples, right? it isn't clear I think, maybe we can add some comments to clearly delineate the two

also this README doesn't feel quite right, what is `file_stream`? maybe this should be like
```
bytes_io = BytesIO(b'This is a sample file.')
```